MultipleFileUpload/Model/IQueue.php

```diff
@@ -72,7 +72,7 @@ function addFileManually($name, $chunk, $chunks);
 	 * @param type $chunk
 	 * @param FileUpload $file
 	 */
-	function updateFile($name, $chunk, FileUpload $file = null);
+	function updateFile($name, $chunk, FileUpload $file);
```

Comment:
I put this question conversely: is there special reason to have optional parameter?